packages/manager/modules/pci/src/projects/projects.constant.js

```diff
@@ -25,6 +25,7 @@ export const PCI_FEATURES = {
     DATA_PROCESSING: 'data-processing',
     LOGS_DATA_PLATFORM: 'logs-data-platform',
     HORIZON: 'horizon',
+    PUBLIC_GATEWAYS: 'public-cloud:public-gateways',
```

Comment:
Question, do you know if `public gateway` product can be or will be use in another universe part ?

Reply:
I am not sure about another universe, but as per discussion it should be there for public cloud.

Reply:
Not required to move it in another place.
My point is, if the `public gateway` can be used in others universes, so you have to remove the `public-cloud` prefix and create a dedicated feature-flipping file, if not so it's good.